test/nms-openwire-test.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Can we make multi-target, see https://github.com/apache/activemq-nms-api/blob/master/src/nms-api/nms-api.csproj

Reply:
I think it makes sense to multi-target the main project, but why the test project? If we multi-target the main binaries, do I need to multi-target the test project simply so the test project can run against all the possible frameworks of the main project?

Reply:
yes, idea would be to test using core(2) and framework (3.5,4)

Reply:
What is there to this? I changed to `TargetFrameworks` and added net35 and net40 but the solution wont compile now. I also added some itemgroup conditions I saw in the API solution.

![image](https://user-images.githubusercontent.com/660420/62588298-df212600-b88a-11e9-8e93-f07d6fa8cef3.png)